localization/english/event_localization/wc_scarlet_events_l_english.yml

```
@@ -0,0 +1,72 @@
+﻿l_english:
+ SCARL.000_TITLE:1 "Those that Remain"
+ SCARL.000_DESC:0 "Once a beacon of the Light, the [GetTitleByKey('k_lordaeron').GetName] is now little more than a corpse-ridden ruin, a feast for the damned. Those of us who yet live are few, scattered and hunted by the Scourge and worse yet — traitors who once called themselves our brethren who have willingly submitted themselves into the armies of the damned.\n\n[GetTitleByKey('k_order_of_the_silver_hand').GetName] stands on the edge of ruin, its greatest champions slain or turned against us.\n\n[scoped_betrayer.GetName] has delivered our lands to death itself, and now whispers tell of others within our ranks who would rather cower or consort with the enemy than stand and fight.\n\nBut I will not falter.\n\nIf there are still righteous warriors left in this land, I will find them. We will rise from the ashes of Lordaeron, not as victims, but as those who would purge the plague that stole our home. The [GetTrait('lich_king').GetName(GetNullCharacter)] will know that this land belongs to us."
+ SCARL.000_OPT_A:0 "I will send out emissaries to those loyal to the throne."
+ SCARL.000_OPT_A_tt:0 "You will send out calls within [GetTitleByKey('k_lordaeron').GetName] for a joint alliance against the Lich King."
+ SCARL.000_OPT_B:0 "I cannot afford to chase ghosts."
+ SCARL.000_OPT_tt:0 "You choose to stand alone."
+
+ SCARL.001_TITLE:1 "Those that Remain"
+ SCARL.001_DESC:0 "To the last true knights of Lordaeron,\n\nLordaeron has fallen, but the Light still burns in us. The traitors and the damned would see our kingdom lost forever.\nI refuse to yield.\nIf you still have the will to fight, stand with the last of the Silver Hand. Together, we will cleanse this land of its plague and take back what is ours."
+ SCARL.001_OPT_A:0 "For Lordaeron! We stand and fight!"
+ SCARL.001_OPT_A_TT:0 "You will join a meeting with the Silver Hand to discuss the fate of Lordaeron."
+ SCARL.001_OPT_B:0 "This war is already lost."
+ SCARL.001_OPT_B_TT:0 "You ignore the call."
+
+ SCARL.002_TITLE:1 "Those that Remain"
+ SCARL.002_DESC:0 "Your call has been heard. The Light has not abandoned us, nor will we abandon you. Lordaeron may be in ruins, but our duty still remains until our death. We will gather our forces and attend your meeting.\nFor Lordaeron, and for the Light."
+ SCARL.002_OPT_A:0 "Excellent."
+
+ SCARL.003_TITLE:1 "Those that Remain"
```

```suggestion
 SCARL.002_TITLE:1 "$SCARL.001_TITLE$"
```
or you could just use the same loc key for all three titles (might help prevent hash collisions)